docs/topics/time.md

```diff
@@ -0,0 +1,373 @@
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    
+    // Print in minutes with 2 decimal places
+    println(fiveSeconds.toString(DurationUnit.MINUTES, 2))
+    // 0,08m
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To print the `Duration` as an [ISO-8601-compatible](https://en.wikipedia.org/wiki/ISO_8601) string, use the [`toIsoString()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/to-iso-string.html)
+function:
+
+```kotlin
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    println(86420.seconds.toIsoString()) // PT24H0M20S
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Convert duration
+
+To express your `Duration` in a different `Duration.Unit` you can use the properties:
+* `inWholeNanoseconds`
+* `inWholeMicroseconds`
+* `inWholeSeconds`
+* `inWholeMinutes`
+* `inWholeHours`
+* `inWholeDays`
+
+For example:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val thirtyMinutes: Duration = 30.minutes
+    println(thirtyMinutes.inWholeSeconds)
+    // 1800
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+Alternatively, you can use your desired `Duration.Unit` as a function parameter in the following extension functions:
+* `toInt()`
+* `toDouble()`
+* `toLong()`
+
+For example:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    val thirtyMinutes: Duration = 30.minutes
+    println(thirtyMinutes.toDouble(DurationUnit.SECONDS))
+    //1800.0
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Compare duration
+
+To check if `Durations` are equal, use the equality operator (`==`):
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.hours
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val thirtyMinutes: Duration = 30.minutes
+    val halfHour: Duration = 0.5.hours
+    println(thirtyMinutes == halfHour)
+    // true
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To compare `Durations`, use the comparison operators (`<`, `>`):
+
+```kotlin
+import kotlin.time.Duration.Companion.microseconds
+import kotlin.time.Duration.Companion.nanoseconds
+
+fun main() {
+    //sampleStart
+    println(3000.microseconds < 25000.nanoseconds)
+    // false
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Break duration into components
+
+To break down a `Duration` into its time components and perform a further action, use the `toComponents()` function.
```

Comment:
"use one of overloads of `toComponents` function." This requires an explanation how these overloads differ from each other.